docs/configuration.rst

```diff
@@ -325,6 +349,50 @@ Removes a variant set from a given data repository and dataset.
 
     $ ga4gh_repo remove-variantset path/to/datarepo aDataset aVariantSet
 
++++++++++++++++
+add-biosample
++++++++++++++++
+
+Add a BioSample record to the given data repository and dataset. Records
+should be described in JSON format according to the GA4GH protocol. To 
+associate this BioSample with an Individual record add the optional 
+individualName flag.
```

Comment:
The uniqueness restrictions is fine for now.  The API doesn't
require the name to be unique, however it is reasonable for
implementation to place reasonable data restrictions.

There was objection to allowing name to be non-unique, although
this was never resolved.


David Steinberg <notifications@github.com> writes:
> In docs/configuration.rst:
> 
> > @@ -325,6 +349,50 @@ Removes a variant set from a given data repository and dataset.
> >
> >      $ ga4gh_repo remove-variantset path/to/datarepo aDataset aVariantSet
> >
> > ++++++++++++++++
> > +add-biosample
> > ++++++++++++++++
> > +
> > +Add a BioSample record to the given data repository and dataset. Records
> > +should be described in JSON format according to the GA4GH protocol. To
> > +associate this BioSample with an Individual record add the optional
> > +individualName flag.
> 
> This uniqueness is required by the server's architecture, and is not meant to
> reflect the schemas. This name is also the filename of the record with the
> extension removed.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub*
>